README.md

```diff
@@ -31,6 +32,11 @@ Finding aids for textual and multimedia [primary sources](https://en.wikipedia.o
 - [Österreichische Mediathek](https://www.mediathek.at/) - The Austrian archive for sound recordings and videos from cultural and contemporary history.
 - [Wien Geschichte Wiki](https://www.geschichtewiki.wien.gv.at/Wien_Geschichte_Wiki) - It is based on the Historical Dictionary Vienna by Felix Czeike, currently there are 50,156 contributions and 12,510 images.
 
+### Brazil
+
+- [Hemeroteca Digital Brasileira](https://bndigital.bn.gov.br/hemeroteca-digital/) - The Fundação Biblioteca Nacional offers its users the HEMEROTECA DIGITAL BRASILEIRA, a portal of national periodicals that provides wide consultation, through the internet, of its collection of periodicals – newspapers, magazines, yearbooks, bulletins, etc. – and serial publications.
```

Comment:
1. Please make the description as short as possible. (one sentence, no redundancy)
2. Please answer these questions:
> Describe why this change is made. Alternatively, refer to existing issues if
> any. You could try to answer:
> 
> Why do / don't you find this tool awesome?
> How do you use this tool in your workflow?
> What advantages / disadvantages does it have compared to {another tool}?
> I'm just contributing to the addition of a section referring to Brazil and its works in digital history.